c2rust-analyze/src/main.rs

```diff
@@ -361,6 +362,109 @@ impl<'tcx> Debug for AdtMetadataTable<'tcx> {
     }
 }
 
+fn label_rvalue_tys<'tcx>(acx: &mut AnalysisCtxt<'_, 'tcx>, mir: &Body<'tcx>) {
+    for (bb, bb_data) in mir.basic_blocks().iter_enumerated() {
+        for (i, stmt) in bb_data.statements.iter().enumerate() {
+            let (_, rv) = match &stmt.kind {
+                StatementKind::Assign(x) => *x.clone(),
+                _ => continue,
+            };
+
+            if acx.c_void_casts.should_skip_stmt(Location {
+                statement_index: i,
+                block: bb,
+            }) {
+                continue;
+            }
```

Comment:
Why is this part added?  Was this something missed earlier?

Reply:
the thing this was checking before happened downstream of a check of this kind. now it's moved to upstream of that check so another check needed to be added